src/deploy_inst.cc

```diff
@@ -7,7 +7,7 @@ DeployInst::DeployInst(cyclus::Context* ctx)
     : cyclus::Institution(ctx),
       latitude(0.0),
       longitude(0.0),
-      coordinates(latitude, longitude) {}
+      coordinates(0,0) {}
```

Comment:
Should this be updated to match the last commit to the cyclus repo where these are initialized in the header/snippet?